packages/plugins/documentation/server/utils/builders/build-api-requests.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const cleanSchemaAttributes = require('../clean-schema-attributes');
+
+/**
+ *
+ * @param {object} attributes - The attributes found on a contentType
+ * @param {object} route - The current route
+ *
+ * @returns The Swagger requestBody
+ */
+module.exports = (attributes, route) => {
+  const requiredAttributes = Object.entries(attributes)
+    .filter(([, val]) => {
+      return val.required;
+    })
+    .map(([attr, val]) => {
+      return { [attr]: val };
+    });
```

Comment:
ℹ️ I decided to only list required attributes in the request body if there are any...